src/Simulation/Enrollment.jl

```diff
@@ -0,0 +1,142 @@
+module Enrollment
```

Comment:
Snake case the variables & functions in this module

Reply:
Done. 179c41190af4e3708a1c12d1494416deb9d7eb74